ipython_magic/trino/schema_export.py

```diff
@@ -1,46 +1,52 @@
 import json
 import os.path as path
 import time
-
+from trino.exceptions import TrinoUserError
 
 MAX_RET = 20000
 
 def getColumns(cur, tableName):
-    sql = f'SHOW COLUMNS IN {tableName}'
-    #print(sql)
-    cur.execute(sql)
-    rows = cur.fetchmany(MAX_RET)
-    return list(map(lambda r: {
-        'columnName': r[0],
-        'metadata': r[2],
-        'type': r[1],
-        'description': r[3]
-    }, rows))
-
+    try:
+        sql = f'SHOW COLUMNS IN {tableName}'
+        #print(sql)
+        cur.execute(sql)
+        rows = cur.fetchmany(MAX_RET)
+        return list(map(lambda r: {
+            'columnName': r[0],
+            'metadata': r[2],
+            'type': r[1],
+            'description': r[3]
+        }, rows))
+    except TrinoUserError:
+        return []
```

Comment:
I would print/log the error here and below but I was also wondering if you had a chance to test the code cleanup I added back to the 'print_control_chars' branch? We should probably take care of that before making messy merge conflicts.

Reply:
good idea, I added a print statement

Reply:
no I did not checked the print_control_chars again, can you make a PR so I see the changes